service/handlers.go

```diff
@@ -1059,40 +1059,45 @@ func (h handler) deleteToken(w http.ResponseWriter, r *http.Request) {
 	// check if token exists in CP and DB
```

Comment:
referring to lines 1055-1057: is this still the desired behavior for missing projects when deleting a token? what gets returned to the client when we just return/exit the handler function like this? an empty 200 response?

Reply:
Actually, for this case, `projectExists()` returns a 404.